3scale-operator/tasks/main.yml

```diff
@@ -29,29 +37,73 @@
           metadata:
             name: apimanager-sample
           spec:
-            wildcardDomain: 3scale.${CLUSTER_BASE_URL}
+            wildcardDomain: "3scale.{{ cluster_base_url }}"
             system:
               fileStorage:
                 simpleStorageService:
                   configurationSecretRef:
                     name: aws-auth
+        wait: true
+        wait_condition:
+          status: "True"
+          type: "Available"
+        # Usually the APIManager takes 2-3min to get the True status.
+        wait_sleep: 10
+        wait_timeout: 500
+      register: result
+    - name: Create Secret in target_namespace for APICast
+      kubernetes.core.k8s:
+        state: present
+        namespace: "{{ target_namespace }}"
+        definition:
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: 3scaleportal
+          stringData:
+            AdminPortalURL: "https://access-token@3scale-admin.3scale.{{ cluster_base_url }}"
```

Comment:
Update to `"https://access-token@3scale-admin.3scale.apps.{{ cluster_base_url }}"`